src/reyna/reyna.csproj

```diff
@@ -17,7 +17,7 @@
     <PlatformFamilyName>WindowsCE</PlatformFamilyName>
     <PlatformID>E2BECB1F-8C8C-41ba-B736-9BE7D946A398</PlatformID>
     <OSVersion>5.00</OSVersion>
-    <DeployDirSuffix>reyna</DeployDirSuffix>
+    <DeployDirSuffix>elemez</DeployDirSuffix>
```

Comment:
"The node is only there for Smart Device projects.
It sets the directory when deploying to a device via USB (or other means). You can find this setting via project properties, Device menu, Output file folder field."

Could be left blank.